src/Models/BaseElement.php

```diff
@@ -295,6 +295,10 @@ public function getCMSFields()
                     )
             );
 
+            // Rename the "Settings" tab
+            $fields->fieldByName('Root.Settings')
+                ->setTitle(_t(__CLASS__ . '.SettingsTabLabel', 'Settings'));
```

Comment:
Ok, what was the label originally? Is this just so we can make the label translatable with an explicit translation key?

Reply:
The default "Settings"; I added the key to https://github.com/dnadesign/silverstripe-elemental/pull/578/files#diff-e0c0c7c5f72bb31c02647d33f6a0b563R17 according to the other label for "Content"

Is there a more elegant way for translating tab labels?